twitter-lm.html

```diff
@@ -0,0 +1,571 @@
+          consequatur? Voluptatem repellat eveniet placeat eligendi, ullam, 
+          rerum similique temporibus sunt saepe iusto distinctio sed commodi.
+      </p>
+      <div class="tweet-options mg-t-md font-s-rg">
+        <div class="tweet-option flex gap-sm">                           
+            <span>35</span>
+        </div>
+        <div class="tweet-option flex gap-sm">
+            <span>245</span>
+        </div>
+        <div class="tweet-option flex gap-sm">
+            <span>465</span>
+         </div>
+      </div>
+  </div>
+</main>
+
+        <!-- OLD
+          <aside class="right-side">
+          <div class="header-question text-heading cl-fade">
+            <h4 class="right-title">What's Happening...</h4>
+            <br/>
+            <div class="news-container">
+              <article class="border-blue">
+                <div class="news-header two-grid-column__expand-one">
+                  <p>World News <strong>Trending</strong></p>
+                </div>
+                <h3>Russia & Ukraine Latest...</h3>
+                <p>96k tweets</p>
+                <div class="header-reply">Join the conversation...</div>
+              </article>
+              <br/>
+              <article class="border-blue">
+                <div class="news-header two-grid-column__expand-one">
+                  <p>Football <strong>Trending</strong></p>
+                </div>
+                <h3>Mbappe seals 1-0 win for PSG...</h3>
+                <p>82k tweets</p>
+                <div class="header-reply">Join the conversation...</div>
+              </article>
+              <br/>
+              <article class="border-blue">
+                <div class="news-header two-grid-column__expand-one">
+                  <p>F1 <strong>Trending</strong></p>
+                </div>
+                <h3>Masi sacked by FIA following investigation...</h3>
+                <p>74k tweets</p>
+                <div class="header-reply">Join the conversation...</div>
+              </article>
+              <br/>
+              <article class="border-blue">
+                <div class="news-header two-grid-column__expand-one">
+                  <p>Finance <strong>Trending</strong></p>
+                </div>
+                <h3>Tesla reaccept payments in BTC...</h3>
+                <p>47k tweets</p>
+                <div class="header-reply">Join the conversation...</div>
+              </article>
+            </div>
+          </div>
+        </aside> -->
+
+        <aside class="right-side">
+          <div class="searchbar">
+            <i class="fa-solid fa-magnifying-glass"></i>
+            <input class="search" type="text" placeholder="Search Twitter" />
+          </div>
+          <div class="news">
+            <div class="title text-md-bold"><h2>What's happening?</h2></div>
+
+            <div class="news-card">
+              <div class="news-card-article">
```

Comment:
Nice use of cards here. Makes the code very readable.

a div with class "news-card-article", should be an article tag?

Reply:
In hindsight yes, it works either way but tagging it as an article would probably be more efficient.